cloudformation/membership-attribute-service.yaml

```diff
@@ -16,23 +16,6 @@ Parameters:
     - m3.medium
     - m3.large
     - m3.xlarge
-    - m3.2xlarge
-    - c3.large
-    - c3.xlarge
-    - c3.2xlarge
-    - c3.4xlarge
-    - c3.8xlarge
-    - g2.2xlarge
-    - r3.large
-    - r3.xlarge
-    - r3.2xlarge
-    - r3.4xlarge
-    - r3.8xlarge
```

Comment:
I don't think we're ever going to use these instance types

Reply:
always good to keep your options open